tests/eas/generic.py

```diff
@@ -329,6 +329,15 @@ class ThreeSmallTasks(_EnergyModelTest):
     """
     Test EAS for 3 20% tasks over 2 seconds
     """
+
+    # The energy estimation for this test is probably not very accurate and this
+    # isn't a very realistic workload. It doesn't really matter if we pick an
+    # "ideal" task placement for this workload, we just want to avoid using big
+    # CPUs in a big.LITTLE system. So use a larger energy threshold that
+    # hopefully prevents too much use of big CPUs but otherwise is flexible in
+    # allocation of LITTLEs.
+    energy_est_threshold_pct = 20
```

Comment:
> Should we adopt a convention to always use CAPITAL_CASE name for hard-coded magic values?

I prefer lower case here because it's actually a variable (modified by some classes)

> this test hardcode 3 small tasks... which will not fit on platforms like pixels for example.

This will work on any platform. The only case it isn't very useful is if you only have 1 LITTLE and its capacity is less than 30% of big (not likely). Even then, the test will figure out that one or more tasks needs to go on big, and things will work fine.